Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/settings/ChanSettings.java

```diff
@@ -109,6 +109,36 @@ public String getKey() {
         }
     }
 
+    public enum ConcurrentFileDownloadingChunks implements OptionSettingItem {
+        One("One chunk"),
```

Comment:
I mention this elsewhere, but you might be able to just use a IntegerSetting for this instead, with a callback to snap it to a multiple of 2, with some maximum.

Reply:
I did it via ListSettingView because I didn't want the user to be able to enter the number by himself. I rewrote it a little bit so now it looks like similar setting - backgroundTimeout.